gammapy/spectrum/models.py

```diff
@@ -1584,3 +1584,101 @@ def evaluate(self, energy, **kwargs):
+        :math:`\sigma`
+
+
+    Examples
+    --------
+    This is how to plot the default `Gaussian` spectral model:
+
+    .. code:: python
+
+        from astropy import units as u
+        from gammapy.spectrum.models import Gaussian
+
+        gaussian = Gaussian()
+        gaussian.plot(energy_range=[0.1, 100] * u.TeV)
+        plt.show()
+    """
+
+    def __init__(self, norm=1E-12 * u.Unit('cm-2 s-1'), mean=1 * u.TeV,
+                 sigma=2 * u.TeV):
+        self.norm = Parameter("norm", norm)
+        self.mean = Parameter("mean", mean)
+        self.sigma = Parameter("sigma", sigma)
+
+        super().__init__([self.norm, self.mean, self.sigma])
+
+    @staticmethod
+    def evaluate(energy, norm, mean, sigma):
+        return norm / (sigma * np.sqrt(2 * np.pi)) * np.exp((energy - mean) ** 2 / (2 * sigma ** 2))
+
+    def integral(self, emin, emax, **kwargs):
+        r"""Integrate Gaussian analytically.
+
+        .. math::
+            F(E_{min}, E_{max}) = \int_{E_{min}}^{E_{max}} \phi(E)dE = \frac{N_0}{2} \left( erf(\frac{E_{max}-
+            \bar{E}}{\sqrt{2} \sigma}) - erf(\frac{E_{min}-\bar{E}}{\sqrt{2} \sigma}) \right)
+
+
+        Parameters
+        ----------
+        emin, emax : `~astropy.units.Quantity`
+            Lower and upper bound of integration range
+        """
+        from scipy.special import erf
+        # kwargs are passed to this function but not used
+        # this is to get a consistent API with SpectralModel.integral()
+        pars = self.parameters
+        u_min = (emin - pars['mean'].quantity) / (np.sqrt(2) * pars['sigma'].quantity)
+        u_max = (emax - pars['mean'].quantity) / (np.sqrt(2) * pars['sigma'].quantity)
+
+        int = pars['norm'].quantity / (2) * (erf(u_max) - erf(u_min))
+        return int
+
+    def energy_flux(self, emin, emax):
+        r"""Compute energy flux in given energy range analytically.
+
+        .. math::
+            G(E_{min}, E_{max}) = \int_{E_{min}}^{E_{max}}E \phi(E)dE =
+            \frac{N_0 \sigma}{\sqrt{2*\pi}}* \left( \exp(\frac{E_{min}-\bar{E}}{\sqrt{2} \sigma}) -
+            exp(\frac{E_{max} - \bar{E}}{\sqrt{2} \sigma}) \right) \\ + \frac{N_0 * \bar{E}}{2} \left( erf(\frac{E_{max}-\bar{E}}{\sqrt{2} \sigma})
+            - erf(\frac{E_{min}-\bar{E}}{\sqrt{2} \sigma}) \right)
+
+        Parameters
+        ----------
+        emin, emax : `~astropy.units.Quantity`
+            Lower and upper bound of integration range.
+        """
+        from scipy.special import erf
+
+        pars = self.parameters
+        u_min = (emin - pars['mean'].quantity) / (np.sqrt(2) * pars['sigma'].quantity)
+        u_max = (emax - pars['mean'].quantity) / (np.sqrt(2) * pars['sigma'].quantity)
+        a = pars['norm'].quantity * pars['sigma'].quantity / np.sqrt(2 * np.pi)
+        b = pars['norm'].quantity * pars['mean'].quantity / 2
+        int = a * (np.exp(- u_min ** 2) - np.exp(-u_max ** 2)) + b * (erf(u_max) - erf(u_min))
```

Comment:
int is a keyword, shouldn't be used a local variable name. Just change `int = ` to `return` here.